landau/plot.py

```diff
@@ -362,26 +362,70 @@ def plot_1d_mu_phase_diagram(df):
     plt.xlabel("Chemical Potential Difference [ev]")
     plt.ylabel("Semi-grandcanonical Potential [eV/atom]")
 
-def plot_1d_T_phase_diagram(df):
-    """Plot a one dimensional equipotential phase diagram."""
+def plot_1d_T_phase_diagram(
+        df, 
+        ax=None, 
+        show=True, 
+        mark_transitions=True,
+        return_Tt=False):
+    """
+    Plot a one dimensional equipotential phase diagram as a function of temperature.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        Input data containing columns for temperature ('T'), potential ('phi'),
```

Comment:
```suggestion
    Args:
        df (pandas.DataFrame):
            Input data containing columns for temperature ('T'), potential ('phi'),
```

We use [google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) in pyiron usually.  (It's less typing too :p)